src/name_service/core/use_case.clj

```diff
@@ -10,17 +10,61 @@
       (action/make-action type payload))))
 
 (defn add-key-binding [{:keys [storage]}]
-  "Add value 'val' for key 'key'."
-  (fn [key val]))
+  "Add value A for key X."
+  (fn [key val]
+    (let [keymap {key val}]
+      (result->action
+        :entity/add
+        (entity/save storage keymap nil nil)))))
+
+(defn append-key-binding [{:keys [storage]}]
+  "Add a value A for key X, where key Y has value B."
```

Comment:
In the documentation refer to argument names so that it's clear which is which (`key-a` is key `X` and so on).

Reply:
Additionally, document what this function returns. Especially mention possible types of results and errors (`:entity/add` or error `:key-exists`).